sdk/nodejs/__tests__/cloudClient.spec.ts

```diff
@@ -251,9 +254,10 @@ describe('DVCCloudClient with EdgeDB Enabled', () => {
             expect(res).toEqual({})
         })
 
-        it('to return no features when allFeatures call fails', async () => {
-            const res = await client.allFeatures(badUser)
-            expect(res).toEqual({})
+        it('throws exception when allFeatures called with invalid user', async () => {
+            await expect(async () => {
+                await client.allFeatures(badUser)
+            }).rejects.toThrow('Bucketing request failed with status 401.')
```

Comment:
this shouldnt be a 401 if the only problem is an invalid user

Reply:
this is just because the mock server returns 401 when the user id is 'bad user', but the actual api returns 400 so i guess it makes sense to match